src/lambda/commands/createNewSamApp.ts

```diff
@@ -51,12 +51,17 @@ import { launchConfigDocUrl } from '../../shared/constants'
 import { Runtime } from 'aws-sdk/clients/lambda'
 import { getIdeProperties } from '../../shared/extensionUtilities'
 
+type CreateReason = 'unknown' | 'userCancelled' | 'fileNotFound' | 'complete' | 'error'
```

Comment:
Do we want to add a CreateReason for resumption? e.g. `completeResumed` and `errorResumed`?

Reply:
If it can be added without a custom vscode-only telemetry definition then that seems useful.